### DISCUSSION TITLE
Changing global security settings without be logged in at the database / any database

### DISCUSSION THREAD

**Discussion by Roheed05** [keepassxreboot/keepassxc#10210] [Ideas]
Hello together, 

i think it ist not a good idea to be able to editing global security settings without be logged in at any database.

for example you have auto-log off time set to 30 seconds or log off when minimized. You are used to KPXC behaving like this. Now, imagine you use the program in an open office space. If you forget to lock your PC for 2 minutes when you're away, anyone can come to your computer and change the auto-log-off settings arbitrarily. Then, the next time you leave your workplace and simply minimize KPXC out of habit, thinking it will lock, nothing happens, and your colleague gains access to your entire database.

It would also be beneficial if these global security settings could be saved per database. For instance, you might have a database with less critical information that can be configured differently than a more sensitive database.

What do you think about?

**Comment by varjolintu** [comment]
There's one PR that already adresses this: https://github.com/keepassxreboot/keepassxc/pull/9406. It allows moving any database related settings to the database itself instead of the INI file. The same implementation could be used for other global settings in the future, if user wishes to keep the options secure.
